xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/macros.vm

```diff
@@ -489,6 +489,21 @@ $displayName##
  *#
 #macro(livetablecallback $divid $collist $colprops $transprefix)
 function (row, i, table) {
+   ## Make sure the columns containing HTML are formatted in HTML indeed
+   #set ($htmlColumnKeys = ['doc.location'])
+   #foreach($htmlColumnKey in $htmlColumnKeys)
+     #if ($collist.contains($htmlColumnKey))
+       #set ($htmlColumnProps = $colprops[$htmlColumnKey])
+       #if (!$htmlColumnProps)
+         #set ($htmlColumnProps = {})
+       #end
+       #if ($htmlColumnProps.html == $NULL)
+         #set ($htmlColumnProps.html = true)
+       #end
+       #set ($colprops[$htmlColumnKey] = $htmlColumnProps)
```

Comment:
This is needed only if the ``$htmlColumnProps`` was not already set, so in the ``#if (!$htmlColumnProps)`` above. Otherwise looks good.

Reply:
Thanks Marius, I updated the file accordingly.